storybookjs/storybook
# docs: add basic story snippets for Vue

Issue: Nope 😄 

## What I did

I've added some snippets for Vue to the "Writing stories" documentation.

## How to test

:warning: I wasn't able to run this locally (see https://github.com/storybookjs/storybook/issues/11986), so I hope this works? 🤞 🙏
